src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -187,7 +187,7 @@ export type AvatarProject = {
 /**
  * Simple timeseries data used in groups, projects and release health.
  */
-export type TimeseriesValue = [timestamp: number, value: number];
```

Comment:
Why remove this?

Reply:
My IDE was giving me a warning but I realize that this is valied TS. I'll put it back

Reply:
your TS version is probably outdated in your IDE